File: tests/fixtures/cpp/basic.cpp
// expect: total=14 code=6 comment=4 blank=4 shebang=0
// simple c++ fixture

#include <iostream>

/* block comment start
still comment */

int main() {
	int x = 1; // inline comment
	std::cout << "hi" << std::endl;
	return x;
}

File: tests/fixtures/typescript/basic.ts
// expect: total=12 code=5 comment=3 blank=4 shebang=0
// simple typescript fixture

/* block comment */

type Point = { x: number; y: number };
function add(a: number, b: number): number {
	return a + b;
}

const origin: Point = { x: 0, y: 0 };
